e2e/cockroach/cockroach.go

```diff
@@ -37,7 +38,7 @@ func (c *Node) Start(ctx context.Context) error {
 	cmd := []string{
 		"./cockroach",
 		"start",
-		"--store=node" + c.ID,
+		"--store=type=mem,size=640MiB",
```

Comment:
this seems to have had the largest effect; I haven't seen a failure since changing this